ignite/internal/analytics/analytics.go

```diff
@@ -52,23 +56,46 @@ func SendMetric(wg *sync.WaitGroup, cmd *cobra.Command) {
 		return
 	}
 
-	path := cmd.CommandPath()
-	met := gacli.Metric{
-		Name:      cmd.Name(),
-		Cmd:       path,
-		Tag:       strings.ReplaceAll(path, " ", "+"),
-		OS:        runtime.GOOS,
-		Arch:      runtime.GOARCH,
-		SessionID: dntInfo.Name,
-		Version:   version.Version,
-		IsGitPod:  gitpod.IsOnGitpod(),
-		IsCI:      getIsCI(),
+	versionInfo, err := version.GetInfo(context.Background())
+	if err != nil {
+		return
+	}
+
+	var (
+		path         = cmd.CommandPath()
+		scaffoldType = ""
+	)
+	if strings.Contains(path, "ignite scaffold type") {
```

Comment:
what is this weird case? 😅

Reply:
nice catch, I don't know, maybe some test or merge issues, I will remove it, thanks